src/index.js

```diff
@@ -53,6 +56,8 @@ export default (customConfiguration = {}) =>
             margin,
         } = config;
 
+        const getEvent = () => d3.event; // keep d3.event mutable see https://github.com/d3/d3/issues/2733
```

Comment:
I prefer the way enounced here: https://github.com/d3/d3/issues/2733#issuecomment-190743489

Reply:
will not work with a global d3